pkg/proxy/conntrack/conntrack.go

```diff
@@ -0,0 +1,103 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package conntrack
+
+import (
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/klog/v2"
+	"k8s.io/kubernetes/pkg/proxy"
+	utilproxy "k8s.io/kubernetes/pkg/proxy/util"
+	"k8s.io/kubernetes/pkg/proxy/util/conntrack"
+	utilexec "k8s.io/utils/exec"
+)
+
+func CleanStaleConnections(isIPv6 bool, exec utilexec.Interface, svcPortMap proxy.ServicePortMap,
```

Comment:
this is meant to replace the duplicate codev on the proxies, no?
if that is the case you should do the change in an additional commit

Reply:
@aojea yeah, this function will replace the dup code.
Wanted a review before pushing the final change. Will push it by EOD.